docs/sources/send-data/fluentbit/fluent-bit-loki-tutorial.md

```diff
@@ -0,0 +1,268 @@
+
+1. To get started, clone the repository that contains our demo application:
+   
+    ```bash
+    git clone -b fluentbit-official  https://github.com/grafana/loki-fundamentals.git
+    ```
+
+1.  Next we will spin up our observability stack using Docker Compose:
+
+    ```bash
+    docker compose -f loki-fundamentals/docker-compose.yml up -d
+    ```
+
+    This will spin up the following services:
+    ```console
+    ✔ Container loki-fundamentals-grafana-1       Started                                                        
+    ✔ Container loki-fundamentals-loki-1          Started                        
+    ✔ Container loki-fundamentals-fluent-bit-1    Started
+    ```
+Once we have finished configuring the Fluent Bit agent and sending logs to Loki, we will be able to view the logs in Grafana. To check if Grafana is up and running, navigate to the following URL: [http://localhost:3000](http://localhost:3000)
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Step 2: Configure Fluent Bit to send logs to Loki
+
+To configure Fluent Bit to receive logs from our application, we need to provide a configuration file. This configuration file will define the components and their relationships. We will build the entire observability pipeline within this configuration file.
+
+### Open your code editor and locate the `fluent-bit.conf` file
+
+Fluent Bit requires a configuration file to define the components and their relationships. The configuration file is written using Fluent Bit configuration syntax. We will build the entire observability pipeline within this configuration file. To start, we will open the `fluent-bit.conf` file in the code editor:
+
+{{< docs/ignore >}}
+> Note: Killercoda has an inbuilt Code editor which can be accessed via the `Editor` tab.
+1. Expand the `loki-fundamentals` directory in the file explorer of the `Editor` tab.
+1. Locate the `fluent-bit.conf` file in the top level directory, `loki-fundamentals'.
+1. Click on the `fluent-bit.conf` file to open it in the code editor.
+{{< /docs/ignore >}}
+
+<!-- INTERACTIVE ignore START -->
+1. Open the `loki-fundamentals` directory in a code editor of your choice.
+1. Locate the `fluent-bit.conf` file in the `loki-fundamentals` directory (Top level directory).
+1. Click on the `fluent-bit.conf` file to open it in the code editor.
+<!-- INTERACTIVE ignore END -->
+
+You will copy all of the configuration snippets into the `fluent-bit.conf` file.
+
+### Receiving Fluent Bit protocal logs
+
+The first step is to configure Fluent Bit to receive logs from the Carnivorous Greenhouse application. Since the application is instrumented with Fluent Bit logging framework, it will send logs using the forward protocol (unique to Fluent Bit). We will use the `forward` input plugin to receive logs from the application.
+
+Now add the following configuration to the `fluent-bit.conf` file:
```

Comment:
I ran through the tutorial again today, and this time when I opened the `fluent-big.config` file, the [INPUT] snippet was already copied into the file.  I think we want to fix that before we merge this PR.